trio/_core/_tests/test_guest_mode.py

```diff
@@ -505,11 +540,11 @@ async def crash_in_io(in_host):
 
     with monkeypatch.context() as m:
 
-        async def crash_in_worker_thread_io(in_host):
+        async def crash_in_worker_thread_io(in_host: InHost) -> None:
             t = threading.current_thread()
             old_get_events = trio._core._run.TheIOManager.get_events
 
-            def bad_get_events(*args):
+            def bad_get_events(*args: Any) -> object:
```

Comment:
Any reason not to use `object` instead of `Any`?

Reply:
I'm pretty sure you'd get a typing error on line 516/551 in the `old_get_events(*args)` call. The high-effort solution (here and elsewhere in this PR) would be to copy the signature of it and not use `*args` at all.